PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1708,7 +1708,7 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 				enableFindDlgItem(IDREDOTMATCHNL, false);
 			}
 			enableProjectCheckmarks();
-			return TRUE;
+			return FALSE;

```

Comment:
Though the value of FALSE is also 0, I think it'll be better use 0 instead, for the semantic reason.
As well, in the online document: `If an application processes this message, it should return zero.`
https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-activate